handlers/external/external.go

```diff
@@ -300,12 +300,24 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 			"channel":      msg.Channel().UUID().String(),
 		}
 
+		// put quick replies on last message part, converting from array to string
+		if i == len(parts) - 1 {
+			form["quick_replies"] = buildQuickRepliesResponse(msg.QuickReplies(), sendMethod, contentType)
+		}
+
 		// if we are smart, first try to convert to GSM7 chars
 		if encoding == encodingSmart {
 			replaced := gsm7.ReplaceSubstitutions(part)
 			if gsm7.IsValid(replaced) {
 				form["text"] = replaced
 			}
+
+			if len(msg.QuickReplies()) > 0 {
+				replaced := gsm7.ReplaceSubstitutions(form["quick_replies"])
```

Comment:
Not sure this makes any sense to include. Quick replies really aren't going to be used somewhere that only support GSM7 encoding.